src/components/backend-ai-registry-list.ts

```diff
@@ -227,22 +227,22 @@ class BackendAIRegistryList extends BackendAIPage {
    * */
   _addRegistry() {
     // somehow type casting is needed to prevent errors, unlike similar use cases in other files
```

Comment:
Speaking of the type, How about declaring them as a member variable with `@query` decorator and access modifier(perhaps `private`)? I think we don't need to expose them to avoid unexpected value changes outside of the component. And If we convert them to member variable, we don't need to call `querySelector` or `getElementbyId` functions.

Reply:
> How about converting local variables to member variables with an access modifier?

I think it's good! Are you saying that we should change the variables necessary for registry information such as hostname and url into member variables?

Reply:
> Speaking of the type, How about declaring them as a member variable with `@query` decorator and access modifier(perhaps `private`)? I think we don't need to expose them to avoid unexpected value changes outside of the component. And If we convert them to member variables, we don't need to call `querySelector` or `getElementbyId` functions.

I applied it. 
I replaced the variables for registry information with member variables with access modifier(private).